### ISSUE TITLE
IR Transmitter

### ISSUE THREAD

**Issue by soundsnow** [philj56/gbcc-android#41]
The Game Boy Color had an infrared transmitter and receiver. Generation II Pokémon games made use of this feature through Mystery Gift. Super Mario Bros. Deluxe could send high scores to another cartridge.

**Comment by philj56** [comment]
Yup, IR is something I just haven't attempted yet (and is kind of pointless until I get multiplayer working anyway)
